plugins/modules/zos_apf.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# Copyright (c) IBM Corporation 2020, 2022, 2023
+# Copyright (c) IBM Corporation 2020 - 2024
```

Comment:
seems like we've been preferring to use commas for copyright: `2020, 2024`

Reply:
done